tests/functional/testpilot/test_testpilot_login.py

```diff
@@ -3,13 +3,10 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
-from marionette_driver import By, Wait, expected
-
 from apps.testpilot.app import TestPilot
 
 
 class TestTestPilotLogin(object):
 
     def test_user_can_log_in(self, sign_in, puppeteer, firefox, base_url):
-        test_pilot_page = TestPilot(puppeteer, firefox, base_url)
-
+        test_pilot_page = TestPilot(puppeteer, firefox, base_url)  # noqa
```

Comment:
You may want to fetch this and make sure I didn't break literally everything. I'm just making stuff up.